README.md

```diff
@@ -1,307 +1,277 @@
 ## Creates a Development Container for Java on Visual Studio Codespaces
 
-## High level process
-
-![codespacesProcess](https://raw.githubusercontent.com/DataSnowman/EphrataTEALS/master/images/codespacesProcess.png)
-
-
-## Prereqs
+These instructions will guide you through the process of setting up Codespaces, an online IDE. _IDE_ stands for _Integrated Development Environment_, which is a program that software developers use to write code. You can use any text editor to write code, but an IDE makes it _much_ easier.
 
-To create the Codespace, you'll need the following:
+Think of Codespaces like Microsoft Word for code. You can write a really long essay in Notepad if you want to, but you'll almost always prefer to do it in Word or Google Docs.
 
-### Azure for Student Subscription (or existing Azure subscription)
-
-Creating an Azure for Student subscription is a three-step process.
-
-    1. Create a new Outlook.com email account
-    2. Redeem your Student Verification Code
-    3. Activate your subscription
+## High level process
 
-Click [HERE](https://github.com/DataSnowman/EphrataTEALS/tree/master/docs/AzureForStudentVerificationActivation.pdf) to open the AzureForStudentVerificationActivation.pdf which has more details.
+![codespacesProcess](https://raw.githubusercontent.com/DataSnowman/EphrataTEALS/master/images/codespacesProcess.png)
 
-Here are the basic directions if you don't use the pdf.  If you use the AzureForStudentVerificationActivation.pdf then you can skip forward to [GitHub Account - Configure your GitHub](https://github.com/DataSnowman/EphrataTEALS#github-account---configure-your-github)
 
-`Step 1:` Create a new Outlook.com email account:
+### Create an Azure for Students Subscription
 
-Open a new In-Private Browser session and navigate to https://signup.live.com and create a new Outlook.com email account
+Creating an Azure for Students subscription is a three-step process.
 
-* Click on the `Get A New Email Address` button
-* Add in the email address you would like to use - if the username is not
-available Microsoft will offer several suggestions
-* Create a new password for the account
-* Add in your name, country/region and date of birth and validate using
-Captcha
+ 1. Create a new Outlook.com email account
+ 2. Redeem your Student Verification Code
+ 3. Activate your subscription
 
-The account will be created
-* Select More Actions and click View Inbox
+**Step 1: Create a new Outlook.com email account**
 
-`Step 2:` Redeeming a Microsoft Azure Student Verification Code:
+The first step is to create a new Outlook.com email account to use with Codespaces. You will _only_ be using this account for Codespaces.
 
-Open a new In-Private Browser session and navigate to: http://aka.ms/azure4students
+Your email should have the following format: `<firstname>-<lastinitial>-ephrata-teals@outlook.com`. So, for example, if your name was John Doe, your new email would be `john-d-ephrata-teals@outlook.com`.
```

Comment:
It would be nice to come up with a gender neutral name here, or maybe we can reference the school mascot (Tigers) somehow?